.eslintignore

```diff
@@ -1,3 +1,2 @@
 .nyc_output
-node_modules
```

Comment:
No, it does not take .gitignore, but just ignores `node_modules` and `bower_components`